src/cli/et/make/templates/blockchain_test.py.j2

```diff
@@ -0,0 +1,26 @@
+"""
+A blockchain test for [EIP-{{eip_number}} {{eip_name}}](https://eips.ethereum.org/EIPS/eip-{{eip_number}}).
+"""
+
+import pytest
+
+from ethereum_test_tools import Alloc, Block, BlockchainTestFiller, Transaction
+
+REFERENCE_SPEC_GIT_PATH = "DUMMY/eip-DUMMY.md"
```

Comment:
You could actually populate the correct information for `REFERENCE_SPEC_GIT_PATH`, but as mentioned, don't waste too much time as I'd prefer to remove this feature from EEST entirely.

Reply:
Yes. I have created an issue to remove it #963.

Once we merge this, I will reference these lines in the issue.